tests/rare/SpecificUnitTests.toml

```diff
@@ -11,4 +11,4 @@ runSetup=false
     [[test.workload]]
     testName = 'UnitTests'
     maxTestCases = 1
-    testsMatching = 'noSim/'
```

Comment:
Not sure if this is intended or not.

Reply:
This `tests/rare/SpecificUnitTests.toml` is included in simulation, where we don't want noSim.